### PR TITLE
Adding separate installation scripts for MellanoxOFED and NvidiaGPUDr…

### PR DESCRIPTION
**Pull request by anshuljainansja**
…iver to be used in extensions

### CODE REVIEW

File: install_mellanoxofed.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    pushd "ubuntu/ubuntu-18.04-hpc"
+else
+    echo "*** Error - unsupported distro!"
+    exit -1
+fi
+
+./install_mellanoxofed.sh
+popd
+
+exit 0
```
Comment (jithinjosepkl) [inline install_mellanoxofed.sh:32]: What's the purpose of adding these scripts (install_nvidiagpudriver.sh, install_mellanoxofed.sh)? Are you planning to invoke these from the extension handlers? If so, please put these under a directory (say, extension-scripts).

Comment (anshuljainansja) [reply install_mellanoxofed.sh:32]: yes, that's the purpose, I'll move then in.

File: extension_scripts/install_mellanoxofed.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+then
+    echo "Detected distro: ${distro}"
+else
+    echo "*** Error - invalid distro!"
+    exit -1
+fi
+
+if [[ $distro == "CentOS Linux 7.6.1810" ]]
+then
+    pushd "centos/centos-7.x/centos-7.6-hpc"
```
Comment (jithinjosepkl) [inline extension_scripts/install_mellanoxofed.sh:14]: path needs to be updated as well.

File: extension_scripts/install_nvidiagpudriver.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+then
+    echo "Detected distro: ${distro}"
+else
+    echo "*** Error - invalid distro!"
+    exit -1
+fi
+
+if [[ $distro == "Ubuntu 18.04.4" ]]
+then
+    pushd "ubuntu/ubuntu-18.04-hpc"
```
Comment (jithinjosepkl) [inline extension_scripts/install_nvidiagpudriver.sh:14]: same here.

**Comment by jithinjosepkl** [comment]
Thanks!

File: extension_scripts/install_mellanoxofed.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+distro=`./../common/extract_distro.sh`
+if [ $? -eq 0 ]
+then
+    echo "Detected distro: ${distro}"
+else
+    echo "*** Error - invalid distro!"
+    exit -1
+fi
+
+if [[ $distro == "CentOS Linux 7.6.1810" ]]
```
Comment (vermagit) [inline extension_scripts/install_mellanoxofed.sh:12]: I'd imagine the minor version (1810) would have to be stripped off, no?
Same everywhere in this file and tests/run_tests.sh

**Pull request merged**